mw-server/services/way.service.go

```diff
@@ -303,60 +304,80 @@ func UpdateWayIsCompletedStatus(db *dbb.Queries, ctx context.Context, wayUuid uu
 	return err
 }
 
-func GetPlainWayById(db *dbb.Queries, ctx context.Context, wayUuid uuid.UUID) (schemas.WayPlainResponse, error) {
-	way, err := db.GetWayById(ctx, wayUuid)
+func GetPlainWayById(dbPGX *dbbPGX.Queries, ctx context.Context, wayUuid uuid.UUID) (schemas.WayPlainResponse, error) {
+	wayPgUUID := pgtype.UUID{Bytes: wayUuid, Valid: true}
+	wayDetails, err := dbPGX.GetWayDetailsById(ctx, wayPgUUID)
+	if err != nil {
+		return schemas.WayPlainResponse{}, err
+	}
+
+	copiedFromWayUuid := util.MarshalPgUUID(wayDetails.CopiedFromWayUuid)
+
+	mentorsRaw, ok := wayDetails.Mentors.([]interface{})
+	if !ok {
+		return schemas.WayPlainResponse{}, fmt.Errorf("failed to assert type for mentors")
+	}
 
-	copiedFromWayUuid := util.MarshalNullUuid(way.CopiedFromWayUuid)
-	mentorsRaw, _ := db.GetMentorUsersByWayId(ctx, way.Uuid)
+	mentors := lo.Map(mentorsRaw, func(mentor interface{}, i int) schemas.UserPlainResponse {
+		mentorMap := mentor.(map[string]interface{})
+		createdAt, err := time.Parse("2006-01-02T15:04:05", mentorMap["created_at"].(string))
+		if err != nil {
+			fmt.Println("Error parsing created_at:", err)
+		}
 
-	mentors := lo.Map(mentorsRaw, func(dbMentor dbb.User, i int) schemas.UserPlainResponse {
 		return schemas.UserPlainResponse{
-			Uuid:        dbMentor.Uuid.String(),
-			Name:        dbMentor.Name,
-			Email:       dbMentor.Email,
-			Description: dbMentor.Description,
-			CreatedAt:   dbMentor.CreatedAt.Format(util.DEFAULT_STRING_LAYOUT),
-			ImageUrl:    dbMentor.ImageUrl,
-			IsMentor:    dbMentor.IsMentor,
+			Uuid:        mentorMap["uuid"].(string),
+			Name:        mentorMap["name"].(string),
+			Email:       mentorMap["email"].(string),
+			Description: mentorMap["description"].(string),
+			CreatedAt:   createdAt.Format(util.DEFAULT_STRING_LAYOUT),
+			ImageUrl:    mentorMap["image_url"].(string),
+			IsMentor:    mentorMap["is_mentor"].(bool),
```

Comment:
It's very dangerous, looks like we have interface{} here. This means it will brings errors in run time unpredictable for any changes for user response here. Could we solve it somehow?